tams_ur5_setup_bringup/launch/xtion.launch

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<launch>
+	<param name="/camera/driver/data_skip" value="10" />
```

Comment:
I believe we don't need this line anymore. It was used to reduce the amount of images transferred via the network a year ago.